vscode/src/jsonrpc/agent-protocol.ts

```diff
@@ -901,3 +905,15 @@ export interface ProtocolCodeAction {
         | undefined
         | null
 }
+
+/**
+ * Omitting uris parameter will retrieve all open documents for the
+ * current workspace root.
+ */
+export interface GetDocumentsParams {
+    uris?: string[]
```

Comment:
needed to fix protocol.test.ts
```suggestion
    uris?: string[] | undefined | null
```